File: lab1.6/BankAccount/BankAccount.cpp
#include "BankAccount.h"

BankAccount::BankAccount(long newID)
{
    this->ID = newID;
    this->balance = 0;
}
void BankAccount::withdraw(double amount)
{
    if(NO_MONEY_LEFT)
        return;
    else{
        this->balance -= amount;
    }
}
void BankAccount::deposit(double amount)
{
    this->balance += amount;
}

File: lab1_6/BankAccount.java
package lab1_6;

public class BankAccount {
	private	long ID;
	private double balance;
	public BankAccount(long newID)
	{
		ID = newID;
		balance = 0;
	}
	public void withdraw(double amount)
	{
		if(balance > 0)
			balance -= amount;
	}
	public void deposit(double amount)
	{
		balance += amount;
	}
}
